src/components/Book.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+function Book() {
+  const bookStore = {
+    id: 1,
+    title: '',
+    author: '',
+  };
+  return (
+
+    <>
+
+      <div key={bookStore.id}>
+        <h2>{bookStore.title}</h2>
+        <h5>{bookStore.author}</h5>
+      </div>
+
+    </>
```

Comment:
- [ ] Let us set up this **Book** to be ready to receive data from the other **Books** component. Kindly set up this component to accept the book data as a prop and render the `title` and `author` instead.

  For example in the **`Books`** component, you can pass in `title` and `author` props to Book like so:
  ```
  <Book title="book-title" author="my_author" />
  ```
  Then you can refactor this function to something like this:
  ```
  function Book({ title, author }) {
    return (
      <div>
        <h2>{title}</h2>
        <h5>{author}</h5>
      </div>
    );
  }
  ```

  **NOTE:** You may need to validate prop types. You can install the `prop-types` package and visit this site on [Typechecking with prop-types](https://reactjs.org/docs/typechecking-with-proptypes.html)